### ISSUE TITLE
WCAG 2.4.7 page links to wrong SC for Focus Appearance

### ISSUE THREAD

**Issue by TechThomas** [w3c/wcag#3555]
On the page [Focus Visible (Level AA)
](https://www.w3.org/WAI/WCAG22/Understanding/focus-visible.html) there is a link saying "Passing [Focus Appearance (Minimum)](https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance-minimum) would pass this success criterion." It links to a page that shows this as a AA requirement and being 2.4.11

Current:
Wrong page is linked to [Focus Appearance Minimum](https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance-minimum)

Expected:
The page links to the AAA requirement 2.4.13 [Focus Appearance](https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance.html)

**Comment by matt-t-tinkers** [comment]
There's an additional issue here is that, we don't want it to link to Focus Appearance 2.4.13 because that suggests you need to meet a "AAA" criteria to pass an "AA" one.

Non-text contrast has a number of points on how it should be addressed so it could be linked to that. Although there's an issue on that page that I've just spotted and logged: https://github.com/w3c/wcag/issues/3561
